### PR TITLE
Ignore temp schema repeatable migration for rest test

### LINKED ISSUE #7548: Rest Module Doesn't Init Temp Schema for Test Setup
### Description

With the new temp schema, the rest module is missing needed updates to initialize this schema during tests

### Steps to reproduce

Build the rest module with v2 enabled and notice the errors regarding temp schema

### Additional context

_No response_

### Hedera network

other

### Version

v0.96.0

### Operating system

None

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
* ignore temp schema repeatable migration for rest test

**Related issue(s)**:

Fixes #7548 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by jnels124** [comment]
If preferred could do some init setup to initialize temp schema but rest doesn't utilize any of these tables so probably better to just ignore them

File: hedera-mirror-rest/__tests__/integrationDbOps.js [hunk 1]
```diff
@@ -189,15 +189,12 @@ const getCleanupSql = async () => {
 const getMigratedFilename = () => path.join(process.env.MIGRATION_TMP_DIR, `.${process.env.JEST_WORKER_ID}.migrated`);
 
 const getMigrationScriptLocation = (locations) => {
-  if (!isV2Schema()) {
-    return locations;
-  }
```
Comment (jascks) [inline hedera-mirror-rest/__tests__/integrationDbOps.js:194]: You want to copy the files regardless of v1 or v2?

Comment (jnels124) [reply hedera-mirror-rest/__tests__/integrationDbOps.js:194]: yes both versions use the temp schema files under common.

**Review by jascks — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**